samples/oauth_rails_vethealth/config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+wlILgaX5F1IR89EnySh2OFh6J8Q/mKYnlnxBbw0tc3yCHH0B4PD4DY8PIqPsUiV22w85TVBHdFkoeE1aN60BqySJjyrMX2J3FiTAZfRbbzp72x19UmHGbzWVkZ1pEKQaxV9hzNnqdr62Pn4+G+nGNSjW+lv45sgqm/q3IAG7ocj4WzQPOCEZozcceyXuC1lsZGEbRyYwvN9wj1YEGfxRsa0XazfOB3AGbA48vLIqe29IFZXrgBpdmT6W32sWorYQP3mYCC2NMSwwgFb/Ik75PwDaOYnGIbK/VMFP1L8H5TUr5ZAGScbPaG2ZsF/KxCo2ApgzYDgBQNG8o/vGaWB8IyFcJXF5rVHke2mT8S6rUt4POLGnlQZDyySDRrUFLsWQ3YpKgtFFMx6oIt4Q7LsnDhQ40NQc5NTzpJFH--7oPcVmVOqfnLEgPK--byWMu4pml0Rm4oyau6I1tw==
```

Comment:
is that anyone's super secret token...?

Reply:
I was messing with this https://medium.com/cedarcode/rails-5-2-credentials-9b3324851336 and I think I inadvertently created this file!  I'm going to delete it.